src/WebServer_JSON.ino

```diff
@@ -3,7 +3,41 @@
 #include "src/Globals/Plugins.h"
 #include "StringProviderTypes.h"
 
+// ********************************************************************************
+// Web Interface get RAW value from task
+// ********************************************************************************
+void handle_rawval()
+{
+  String htmlData;
+  const taskIndex_t taskNr    = getFormItemInt(F("tasknr"), INVALID_TASK_INDEX);
+  const bool taskValid = validTaskIndex(taskNr);
+  if (!taskValid)
+  {
+    htmlData += F("ERROR: TaskNr not valid!\n");
+  }
+  const taskVarIndex_t valNr    = getFormItemInt(F("valnr"), INVALID_TASKVAR_INDEX);
+  const bool valueNumberValid = validTaskVarIndex(valNr);
+  if (!valueNumberValid)
+  {
+    htmlData += F("ERROR: ValueId not valid!\n");
+  }
 
+  TXBuffer.startJsonStream();
+  if (taskValid && valueNumberValid)
+  {
+    const deviceIndex_t DeviceIndex = getDeviceIndex_from_TaskIndex(taskNr);
+
+    if (validDeviceIndex(DeviceIndex))
+    {
+      LoadTaskSettings(taskNr);
+      htmlData += String(ExtraTaskSettings.TaskDeviceValueNames[valNr]);
+      htmlData += " ";
```

Comment:
Just as a general hint. When adding a single character, it takes less resources when appending a single char instead of a single character string.
```c++
htmlData += ' ';
```

Reply:
Ahh, Right.
I will fix that, and add option to get all vars in task :)
Will be back to you guys soon :)

Reply:
Maybe get simple CSV formatted output?
Or do you just separate them using a newline?
And not entirely sure if we enforce not using spaces in value names.

Reply:
Good idea with this semicolons.
I think semicolon separating value with name will be better than spaces.
And newline as a separator between variables.
I will change it.

Reply:
So 1 line with only the names and a second one with only the values?
Maybe also have an extra GET parameter then to leave away the names to reduce the resource usage even more.

Reply:
I planned sth like this:
If user sends only tasknr, ex. http://<ip>/raw?tasknr=1 then it will send all variables for selected task
Temperature;23.5
Humidity;80
etc.

It could be possible as uzi18 suggested, to filter only one value, ex:
http://<ip>/raw?tasknr=1&valnr=1
Temperature;23.5

;)